plugins/base/src/main/resources/dokka/templates/includes/header.ftl

```diff
@@ -21,6 +21,9 @@
         <@source_set_selector.display/>
     </div>
     <div class="navigation-controls">
+        <#if homepageLink?has_content>
+            <div class="navigation-controls--btn navigation-controls--homepage" id="homepage-link" role="button"><a href="${homepageLink}"></a></div>
```

Comment:
I have not checked but it seems to be `<a href="${homepageLink}"> <div class="navigation-controls--btn navigation-controls--homepage" id="homepage-link" role="button"/></a>`

Reply:
Im not an html-expert, but at least my variant works. If to switch tags, it doesn't work at my side (may be need to adapt more code)
I think it could be rechecked in scope of #3300